flepimop/gempyor_pkg/src/gempyor/distributions.py

```diff
@@ -476,3 +487,24 @@ def _sample_from_generator(
     | WeibullDistribution,
     Field(discriminator="distribution"),
 ]
+
+DISTRIBUTION_ADAPTER = TypeAdapter(Distribution)
+
+
+def build_distribution_from_confuse_config(config: confuse.ConfigView) -> Distribution:
```

Comment:
```suggestion
def distribution_from_confuse_config(config: confuse.ConfigView) -> Distribution:
```
I think this conveys the same meaning?

Reply:
Nice, sounds good. I guess the word "build" implies something more involved is going on in the function than actually is.